artillery/artillery.yml

```diff
@@ -0,0 +1,123 @@
+config:
+  target: '{{ $processEnvironment.SEND_LEGAL_MAIL_TO_PRISONS_API }}'
+  variables:
+    hmpps_auth_url: '{{ $processEnvironment.HMPPS_AUTH_URL }}'
+    hmpps_system_client_id: '{{ $processEnvironment.HMPPS_SYSTEM_CLIENT_ID }}'
+    hmpps_system_client_secret: '{{ $processEnvironment.HMPPS_SYSTEM_CLIENT_SECRET }}'
+    app_smoketest_lsjsecret: '{{ $processEnvironment.APP_SMOKETEST_LSJSECRET }}'
+    app_smoketest_msjsecret: '{{ $processEnvironment.APP_SMOKETEST_MSJSECRET }}'
+  phases:
+    - duration: 10
+      arrivalRate: 1
+      name: Warm Up
+    - duration: 30
+      arrivalRate: 1
+      rampTo: 20
+      name: Ramp Up
+    - duration: 300
+      arrivalRate: 20
+      name: Sustained Load
```

Comment:
I guess this is the configuration I might want to play with - might be nice to mention this in the readme? And maybe point to the Artillery documentation of this?